crates/coordinate_systems/src/transform.rs

```diff
@@ -0,0 +1,55 @@
+use std::{marker::PhantomData, ops::Mul};
+
+use nalgebra::{AbstractRotation, Isometry, SimdRealField};
+
+use crate::framed::Framed;
+
+#[derive(Debug)]
+pub struct Transform<From, To, Inner> {
```

Comment:
maybe controversial, but I prefer `To` be in front of `From` in the signature, this way transforms are written as
```
L<A> = T<A,B> * T<B,C> * T<C,D> * L<D>
```
instead of
```
L<A> = T<B,A> * T<C, B> * T<D, C> * L<D>
```
For me the first is much easier to grasp. However this also conflicts with our current naming scheme of transformations
`(frame1)_to_(frame2)` (which I also don't like, I'd prefer `(frame2)_from_(frame1)`.

Reply:
You might have to fight with the mathematicians then... This is kinda convention. I would like to keep it this way.

Reply:
I learned it exactly like that in my maths course 😉

Reply:
This still would clash with every naming in the rest of the repo. We should keep it for this PR and start a discussion about naming transforms in general? What do you think?

Reply:
Ok :(